Misc/deployment/az_kubes_setup/holorepo-kubes.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: apps/v1beta1
```

Comment:
For filename, I think `kubernetes` or `k8s` would be more fitting

Reply:
renamed to holorepo-k8s.yaml